src/resmom/stage_func.c

```diff
@@ -571,6 +574,7 @@ remtree(char *dirname)
 		replace(dirname, "\\ ", " ", unipath);
 		forward2back_slash(unipath);
 	} else {
+		log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_FILE, LOG_ERR, __func__, "File not found");
```

Comment:
I feel "File not found" is not appropriate for the situation. instead "Invalid path" or "null path" may be used